activerecord/test/cases/migration/columns_test.rb

```diff
@@ -283,6 +284,62 @@ def test_change_column_default_with_from_and_to
         assert_equal "Tester", TestModel.new.first_name
       end
 
+      def test_change_column_null_false
+        add_column "test_models", "first_name", :string
+        connection.change_column_null "test_models", "first_name", false
+
+        assert_raise(ActiveRecord::NotNullViolation) do
+          TestModel.create!(first_name: nil)
+        end
+      end
+
+      def test_change_column_null_true
+        add_column "test_models", "first_name", :string
+        connection.change_column_null "test_models", "first_name", true
+
+        assert_difference("TestModel.count" => 1) do
+          TestModel.create!(first_name: nil)
+        end
+      end
+
+      def test_change_column_null_with_non_boolean_arguments_raises
+        add_column "test_models", "first_name", :string
+        assert_raise(RuntimeError) do
```

Comment:
Maybe we should also test for the error message, to reduce the possibility of false positives?